### ISSUE TITLE
Can't change the severity of a dotnet_naming_style

### ISSUE THREAD

**Issue by SilverPh0ton** [dotnet/roslyn#36312]
**Version Used**: 
Visual Studio v16.1.1
dotnet v 2.2.300

**Steps to Reproduce**:
Add this to the .editorconfig file:
dotnet_naming_style.private_field_style.capitalization = camel_case
dotnet_naming_style.private_field_style.required_prefix = _
dotnet_naming_style.private_field_style.severity = warning

**Expected Behavior**:
Private field name with no '_' at the beginning should trigger a warning,

**Actual Behavior**:
Private field name with no '_' at the beginning trigger an error,

**Comment by sharwell** [comment]
@SilverPh0ton Does your project have "treat warnings as errors" enabled? Can you post the full content of your .editorconfig file?

**Comment by SilverPh0ton** [comment]
We plan to have a "treat warning as errors" on release mode but even with ```dotnet_naming_style.private_field_style.severity = suggestion```
it doesn't work

**Comment by SilverPh0ton** [comment]
I have made a reop project if you want to take a look to see what's going on https://github.com/SilverPh0ton/Editorconfig_dotnet_naming_style_severity

**Comment by sharwell** [comment]
@SilverPh0ton It turns out the editor is following the severity given in .editorconfig:

```
dotnet_naming_rule.private_fields_should_be_camel_case.severity = error
```

Severity is a property of `dotnet_naming_rule`, not `dotnet_naming_style`.

**Comment by sharwell** [comment]
I'm going to close this as answered, but please consider adding a 👍 vote on #19055 if you are interested in having better editor support for these cases.
